### ISSUE TITLE
Is it possible to animate a Polyline?

### ISSUE THREAD

**Issue by kevando** [airbnb/react-native-maps#800]
Essentially I want to do something like this, and I'm not getting anywhere

`MapView.Polyline.Animated`

Not sure this is even possible...

**Comment by symmetriccurve** [comment]
Do you have example gif to show what you are looking for.

**Comment by kevando** [comment]
Yeah, [here](http://giphy.com/gifs/3oz8xs9inPd1sPN3cA) you can see a `MapView.Marker.Animated` (the missile) move across the screen.

You can see at the very end, I draw a polyline. I was hoping to animate that in the same way I animated the marker.

**Comment by symmetriccurve** [comment]
If I understand correctly you want to have a tail behind the rocket(kinda of smoke tail behind the missile). You can keep adding the coordinates as rocket progresses.

**Comment by kevando** [comment]
haha, no. But that's way cooler than my idea. 
So yes. Something just like that.

**Comment by symmetriccurve** [comment]
😆 🚀 You have to actually update(setState) the Polyline data with a timer

**Comment by kevando** [comment]
This guy gets it!

Yeah I'll fire off a message once I get this another shot.

**Comment by MiLeung** [comment]
I'm using redux to store coordinates for the polyine and it's pretty laggy. I tried making MapView.Polyline an Animated component where each latitude and longitude property of each polyline coordinate was an Animated Value, but it didn't work (wrong proptype; only accepts numbers)

![laggy demo](https://media.giphy.com/media/3ohv4tDm0slIkvDsfS/giphy.gif)

It's a little better on device in release mode, but still a bit laggy. Is this the best I can get performance wise or are there other hacks to decrease the lag?

**Comment by uokesita** [comment]
Can anybody point me in the direction of how to do this animation?
https://www.youtube.com/watch?v=iec4fVTuNCE

I want to replicate the animation of the polilyne (the line with colors blue/yellow/red)
at 0:05 in the youtube channel
